mobify/spline
# Fix Sass deprecation warning re: reverse-gradient function.

Since Sass 3.4.10 (January), passing anything other than a non-string value to the `unquote` function is deprecated. This affects the `reverse-gradient` function, which tries to pass a list through `unquote`. This PR removes this usage of `unquote`.

Status: **Ready to merge**

Reviewers: @jeffkamo @avelinet @cole-sanderson @mlworks @yourpalsonja @nastiatikk 
Ticket: N/A
Linked PRs: N/A

## Changes
- No longer attempt to unquote the final list returned from `reverse-gradient`.

## How to test-drive this PR
- Clone this repo and run `npm install && bower install` to ensure dependencies are installed
- Run `grunt` to compile Sass
- Ensure all the DEBUG statements in output pass
- Open the `test/index.html` file and verify that the reverse-gradient function still works.

## Research resources
- [Sass commit](https://github.com/sass/sass/commit/dc44975d2479fd07a34009b900948b2c4efeb4f8) adding `unquote` deprecation warning
